styx-scheduler-service/src/main/java/com/spotify/styx/monitoring/Stats.java

```diff
@@ -57,5 +57,9 @@
 
   void exitCodeMismatch();
 
+  void registerResourceConfigured(String resource, Gauge<Long> resourceCount);
+
+  void resourceUsed(String resource, long used);
```

Comment:
This can be addressed later when adding proper verbs to other methods. It is a bit weird indeed.